bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/values/StreamValue.java

```diff
@@ -87,7 +90,7 @@ public Type  getIteratorNextReturnType() {
      * @param parent The link to the parent node
      */
     public String stringValue(BLink parent) {
-        return "stream <" + getType().toString() + ">";
+        return getType().toString();
```

Comment:
In cases where ToString is not defined, we seem to be having the basic type prefix, a space, and then some string representation. Should we continue to do the same here?

For example,

```ballerina
import ballerina/io;

public function main() {
    stream<int> st = ([1, 2]).toStream();
    io:println(typeof st);
    io:println(st);
    
    future<()> fn = start func();
    io:println(typeof fn);
    io:println(fn);
}

function func() {

}
```

currently prints

```
typedesc stream<()>
stream <stream<()>>
typedesc future
future {isDone:false}
```

Reply:
Revert: 6dda6a0